bundles/org.openhab.binding.venstarthermostat/src/main/java/org/openhab/binding/venstarthermostat/internal/dto/VenstarAwayModeSerializer.java

```diff
@@ -30,6 +30,10 @@
     public VenstarAwayMode deserialize(JsonElement element, Type arg1, JsonDeserializationContext arg2)
             throws JsonParseException {
         int key = element.getAsInt();
-        return VenstarAwayMode.fromInt(key);
+        try {
+            return VenstarAwayMode.fromInt(key);
+        } catch (IllegalArgumentException e) {
+            return null;
+        }
```

Comment:
These changes don't seem to be JS related. :detective:

Reply:
oof, thanks 🤦